agents/postgres/pgstatstatements/pgstatstatements_test.go

```diff
@@ -295,5 +309,86 @@ func TestPGStatStatementsQAN(t *testing.T) {
 		}
 		expected.Common.Queryid = digests[expected.Common.Fingerprint]
 		tests.AssertBucketsEqual(t, expected, actual)
+		assert.LessOrEqual(t, actual.Postgresql.MBlkReadTimeSum, actual.Common.MQueryTimeSum)
+	})
+
+	t.Run("CheckMBlkReadTime", func(t *testing.T) {
+		r := rand.New(rand.NewSource(time.Now().Unix()))
+		tableName := fmt.Sprintf("customer%d", r.Int())
+		_, err := db.Exec(fmt.Sprintf(`
+		CREATE TABLE %s (
+			customer_id integer NOT NULL,
+			first_name character varying(45) NOT NULL,
+			last_name character varying(45) NOT NULL,
+			active boolean
+		);`, tableName))
+		require.NoError(t, err)
+		defer func() {
+			_, err := db.Exec(fmt.Sprintf(`DROP TABLE %s;`, tableName))
+			require.NoError(t, err)
+		}()
+		m := setup(t, db)
+
+		waitGroup := sync.WaitGroup{}
```

Comment:
That's something I say over and over again: there is no field you can possible initialise in WaitGroup, so please use `var wg sync.WaitGroup`.

And I think we should add go-consistent to this repo.

Reply:
I'll create task for go-consistent